src/modules/auth/types/sign-up.types.ts

```diff
@@ -0,0 +1,7 @@
+export type SignUpStep = 'account-type' | 'credentials';
```

Comment:
I'm not really sure if this type is okay, as in this case we tied up typization with UI representation, I'm not sure it should be in this file, maybe it's better to use it when we actually need it.